### PR TITLE
libtorrent-rasterbar: update to 1.2.2.

### PR DESCRIPTION
**Pull request by daniel-eys**
* qbittorrent-4.2.0 now supports libtorrent-rasterbar 1.2.x (which was
  the only package blocking this update)
* Use `py3_ver` rather than hardcoding python36
* Tested this on x86_64, together with qbittorrent, btfs and deluge, which
  uses libtorrent's python3 binding.

### CODE REVIEW

File: srcpkgs/libtorrent-rasterbar/template [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
 short_desc="C++ bittorrent library by Rasterbar Software"
 maintainer="Eivind Uggedal <eivind@uggedal.com>"
 license="BSD-3-Clause"
 homepage="https://libtorrent.org/"
 distfiles="https://github.com/arvidn/libtorrent/releases/download/libtorrent-${version//./_}/libtorrent-rasterbar-${version}.tar.gz"
-checksum=30040719858e3c06634764e0c1778738eb42ecd0b45e814afa746329a948ead7
+checksum=e579261d7f0acbe82e9b4ce703cb721627cb8075023f8a26405992f489bc6202
+
+case "$XBPS_TARGET_MACHINE" in
+	armv6l*)
```
Comment (pullmoll) [inline srcpkgs/libtorrent-rasterbar/template]: This should (most probably) include `armv5*`, `mips*`, and `ppc|ppc-musl` as well.

Comment (daniel-eys) [reply srcpkgs/libtorrent-rasterbar/template]: Added

**Comment by pullmoll** [comment]
We need to purge libtorrent-rasterbar-python from the repository now as it causes staging and isn't built anymore.

**Comment by pullmoll** [comment]
Or would it help if we add `replaces="libtorrent-rasterbar-python"` to the `libtorrent-rasterbar-python3` subpackage?

**Comment by daniel-eys** [comment]
Yes, we should add `replaces`. I guess this solves staging then.

**Pull request merged**